server/controllers/routes/admin/cohort/deleteCohort.js

```diff
@@ -0,0 +1,17 @@
+const deleteCohort = require('../../../../database/queries/cohort/deleteCohort');
+
+const cohortDelete = async (req, res, next) => {
+  try {
+    await deleteCohort(req.params.cohortId);
+    res.json({
+      StatusCode: 200,
+      data: { message: 'Cohort deleted successfully' },
```

Comment:
@MohammedAlghazali What if cohortId does not exist in the table, and there were no affected rows?  If that case is important to handle (for example, if you want to show an error or a message to that user) then it might be good to check for that case here and return something like 
```
{
  StatusCode: 404, // not found
  data: { message: 'cohort not found' },
}
```

Reply:
yes, there is a problem in this route function, I will solve it